### ISSUE TITLE
Dashboard section of the tutorial raises warning

### ISSUE THREAD

**Issue by tomascsantos** [holoviz/holoviz#303]
#### ALL software version info
(this library, plus any other relevant software, e.g. bokeh, python, notebook, OS, browser, etc)

https://gist.github.com/tomascsantos/815e3ed1a20d56aaa0b042cdf043e40b

#### Description of expected behavior and the observed behavior
I don't think users should see errors in the tutorial without changing anything 

#### Complete, minimal, self-contained example code that reproduces the issue

I just followed tutorial 06_Dashboard
<img width="1733" alt="Screen Shot 2022-05-21 at 8 17 48 PM" src="https://user-images.githubusercontent.com/7690514/169676993-a476ed00-a766-4bf9-bef0-fdce60e690b3.png">

Not a big deal, just trying to catch little things to make a more "professional" tutorial experience :)

**Comment by jbednar** [comment]
Yes! We should definitely address that, caused by an update in one of our dependencies. We'll need to make an overall revision of the tutorial before SciPy, so we should be able to deal with that at that time.
